index.html

```diff
@@ -574,10 +574,13 @@ <h3>Use Cases and Requirements</h3>
         </p>
 
         <p>
-This specification makes no normative statements with regard to the
-conformance of roles in the ecosystem, such as <a>issuers</a>, <a>holders</a>,
-or <a>verifiers</a>, because the conformance of ecosystem roles are highly
-application, use case, and market vertical specific.
+This specification includes both required and optional fields. An <a>issuer</a>
+MUST include all required fields. A <a>verifier</a> SHOULD check that all
+required fields satisfy the normative requirements for each. Optional fields MAY
+be ignored by <a>issuers</a> and <a>verifiers</a>. We make no normative
+statements beyond this with regard to the conformance of roles in the ecosystem
+because the conformance of ecosystem roles is highly application, use case, and
+market vertical specific.
```

Comment:
```suggestion
be ignored by <a>issuers</a> and <a>verifiers</a>. 
```

Take or leave this suggestion: It feels odd to have this statement about no other normative statements. But, I'm good either way as this PR wasn't trying to change that line.

Reply:
I thought about removing that line, but also didn't feel strongly about doing so. If other folks give a thumbs up to the suggestion I'll accept it, otherwise I probably won't.

Reply:
I'm holding the PR until the end of the week to get some thoughts in on the above statement.

Reply:
I support removing it, i can request changes if you like, to make it clearer.